src/main/java/io/confluent/examples/streams/interactivequeries/kafkamusic/KafkaMusicExample.java

```diff
@@ -339,8 +349,10 @@ static KafkaStreams createChartsStreams(final String bootstrapServers,
               aggregate.remove(value);
               return aggregate;
             },
-            topFiveSerde,
-            TOP_FIVE_SONGS_BY_GENRE_STORE
+            Materialized.<String, TopFiveSongs, KeyValueStore<Bytes, byte[]>>as(
+                TOP_FIVE_SONGS_BY_GENRE_STORE)
```

Comment:
Nit: should be in the line above ? (it most places it's formatted like this)

Reply:
if you use the cofluent google style that is how it auto-formats it. It sucks! but that is what it does